tokio-macros/Cargo.toml

```diff
@@ -24,7 +24,7 @@ proc-macro = true
 [dependencies]
 proc-macro2 = "1.0.60"
 quote = "1"
-syn = { version = "2.0", features = ["full"] }
+syn = { version = "2.0", features = ["full", "extra-traits", "visit"] }
```

Comment:
Both extra-traits and visit are not very good from a view of compile-time...

Reply:
Haha, yup.

Reply:
Do you think this is doable without these though? I'm asking more about `visit`, there are workarounds for the `PartialEq` implementation that I'm using from `extra-traits`.